src/core/load-fragment.c

```diff
@@ -3939,7 +3941,14 @@ int config_parse_bind_paths(
                         return 0;
                 }
 
-                s = source;
+                r = unit_full_printf(u, source, &sresolved);
+                if (r < 0) {
+                        log_syntax(unit, LOG_ERR, filename, line, r,
+                                   "Failed to resolved specifiers in \"%s\", ignoring: %m", source);
+                        return 0;
```

Comment:
I couldn't say for sure (again, I'm just working off pattern-matching here, I don't know the codebase), but I expect we're already logging the error message through the `%m` in the `log_syntax`, and the other syntax errors return `0` - not to mention that `ENOMEM` is presumably not the only error that `unit_full_printf` can return.

Are you suggesting I should explicitly test for `-ENOMEM` in the return from `unit_full_printf` and handle it separately, like for the calls to `extract_first_word`?